From: Simon McVittie <smcv@debian.org>
Date: Wed, 5 Aug 2020 11:17:26 +0100
Subject: tests: Send SIGTERM to dbus-daemon

There's no real need to kill it with SIGKILL, and in some configurations
there is cleanup that is only performed when allowed to shut down
gracefully.

Signed-off-by: Simon McVittie <smcv@debian.org>
Forwarded: https://github.com/flatpak/xdg-desktop-portal/pull/521
---
 tests/test-document-fuse.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test-document-fuse.sh b/tests/test-document-fuse.sh
index f0013d4..b889cfc 100755
--- a/tests/test-document-fuse.sh
+++ b/tests/test-document-fuse.sh
@@ -49,7 +49,7 @@ export XDG_RUNTIME_DIR=${TEST_DATA_DIR}/runtime
 cleanup () {
     fusermount -u "$XDG_RUNTIME_DIR/doc" || :
     sleep 0.1
-    kill -9 "$DBUS_SESSION_BUS_PID"
+    kill "$DBUS_SESSION_BUS_PID"
     kill $(jobs -p) &> /dev/null || true
     rm -rf "$TEST_DATA_DIR"
 }
